src/GithubApi.elm

```diff
@@ -15,13 +15,22 @@ contributorsUrl =
 
 requestSettings : Http.Request
 requestSettings =
-    { verb = "GET"
-    , headers =
-        [ ( "Authorization", GithubToken.token )
-        ]
-    , url = contributorsUrl
-    , body = Http.empty
-    }
+    let
+        token =
+            case GithubToken.token of
+                Just token ->
+                    token
+
+                Nothing ->
+                    ""
+    in
+        { verb = "GET"
+        , headers =
+            [ ( "Authorization", token )
+            ]
+        , url = contributorsUrl
+        , body = Http.empty
+        }
```

Comment:
Are your sure we need the authorization header at all? I just removed it and it still works.